src/Altinn.Platform/Altinn.Platform.Storage/Storage.Interface/Altinn.Platform.Storage.Interface.csproj

```diff
@@ -2,8 +2,8 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <OutputType>Library</OutputType>
-    <Version>3.10.0</Version>
-    <AssemblyVersion>3.10.0.0</AssemblyVersion>
+    <Version>3.11.0</Version>
```

Comment:
No need for this change since the Storage.Interface models hasn't changed.

Reply:
Done!